charts/rstudio-connect/templates/NOTES.txt

```diff
@@ -35,6 +42,10 @@ Please consider removing this configuration value.
   {{- end }}
 {{- end }}
 
+{{- if and .Values.launcher.enabled (and (not .Values.sharedStorage.create) (not .Values.sharedStorage.mount) (not $hasLauncherDataDir) (not $hasLauncherDataDirPVCName) ) }}
```

Comment:
TODO: we will need to create an escape hatch to turn this check off in advanced set-ups once storage is an optional add-on (same for the Connect warning)